governance/polls/Ratification Poll - MIP102c2-SP36 - August 12, 2024.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP36) - August 12, 2024
+summary: This subproposal amends MIPs 101, 104, 106, 107, and 113 in preparation for Launch Season.
+discussion_link: https://forum.makerdao.com/t/mip102c2-sp36-mip-amendment-subproposal/24382
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-08-12T16:00:00
+end_date: 2024-08-26T16:00:00
+---
+# Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP36) - August 12, 2024
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, August 12 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP102c2-SP36 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This subproposal seeks to make various updates to the Alignment Artifacts to prepare for Launch Season.*
+
+Please review the links below to inform your position on this proposal before voting.
+
+- [Canonical Proposal Version](https://github.com/makerdao/mips/blob/f7a845a35b74a8e651f48165bc2f8a15dd751b98/MIP102/MIP102c2-Subproposals/MIP102c2-SP36.md)
```

Comment:
We had to update the MIP102c2-SP36 text to fix the rendering of the link on the MIPs Portal, the canonical link now needs to change.

```suggestion
- [Canonical Proposal Version](https://github.com/makerdao/mips/blob/0b14eb18f934a50ac9c8e824aaac127eaeb4080f/MIP102/MIP102c2-Subproposals/MIP102c2-SP36.md)
```